themes/SuiteP/include/SubPanel/tpls/SubPanelTiles.tpl

```diff
@@ -53,7 +53,7 @@
           {/literal}$('.sub-panel .table-responsive').footable();{literal}
           // collapse subpanels when device is mobile / tablet
           if($(window).width() <= SUGAR.measurements.breakpoints.large) {
-            $('.panel-collapse').removeClass('in');
+            $('[id^=subpanel] .panel-collapse').removeClass('in');
```

Comment:
@ojs87 @jack7anderson7 is there a class we can use, instead of the id?

Reply:
the classes for subpanels and regular panels are the same which is why I targetted id's containing subpanel. I can look into adding a class if that would be preferable

Reply:
Hi @ojs87, thank you. If the isn't a class already we can leave it as is